spec/support/test_helper.rb

```diff
@@ -1,24 +1,32 @@
 module DfE
   module Bigquery
     module TestHelper
-      include ::BigqueryStubs
-
-      def stub_bigquery_application_metrics_request(rows: nil, job_complete: true, page_token: nil)
+      def stub_bigquery_application_metrics_request(rows: nil, job_complete: true, page_token: nil, result: true)
```

Comment:
Maybe `result` could have a better name, or we could take another approach?

`result: true` just means that the stubbed response returns values in the response.

Reply:
I tried to think better names but `result` is better than any that I thought.

1.`returns_nil_for_rows: false` as default and force by `returns_nil_for_rows: true` 
2. Create another stub method stub_bigquery_application_metrics_nil_rows or something similar.

I posted the options but `result` still better.